elements/bulbs-video/components/revealed.js

```diff
@@ -274,6 +303,16 @@ export default class Revealed extends React.Component {
     }
   }
 
+  handlePauseEvent (reason) {
+    if (reason.pauseReason === 'external') {
+      return;
+    }
+    else if (reason.pauseReason === 'interaction') {
```

Comment:
yeah, unfortunately this functionality won't work with ads. `jwplayer` does not attach a pause reason to those